### ISSUE TITLE
Enable testing `replaceById()` with inclusion on Cloudant

### ISSUE THREAD

**Issue by agnes512** [loopbackio/loopback-next#4251]
Followup issue from #4148

We have two cases in `repository-tests/../has-many-inclusion-resolver.relation.acceptance.ts` are skip for Cloudant. This is because Cloudant needs the revision property `_rev` to perform replacement operations. Our current test setup has too many workarounds to make all databases happy.

## Acceptance criteria

- [ ] Enable the two skip tests pass on Cloudant
  - [ ] Once #4249 is done, should be easy to define `_rev` dynamically by a factory function.
